nucliadb/nucliadb/export_import/utils.py

```diff
@@ -237,14 +237,14 @@ class ExportStream:
     def __init__(self, export: BytesIO):
         self.export = export
         self.read_bytes = 0
-        self.length = len(export.getvalue())
+        self._length = len(export.getvalue())
 
     async def read(self, n_bytes):
         """
         Reads n_bytes from the export stream.
         Raises ExportStreamExhausted if there are no more bytes to read.
         """
-        if self.read_bytes >= self.length:
+        if self.read_bytes >= self._length:
```

Comment:
can you read *over* ? shouldn't this be `==` ?

Reply:
yup! you're right. changing